### ISSUE TITLE
All Buttons Need Icon Options

### ISSUE THREAD

**Issue by koiinteractive** [elementor/elementor#8845]
## Prerequisites

- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
When setting up buttons in different widgets, it's frustrating that the options for the buttons are inconsistent, more specifically with the icons. For example, there is not a way to natively add an icon to the button in the Call to Action widget. It is also sometimes inconvenient that there isn't a way to control the color and weight of the icons themselves across any button instances.

**Describe the solution you'd like**
It would be great if ALL button instances in ALL widgets had icon options, along with the ability to change their colors, weight, etc.

**Describe alternatives you've considered**
Applying a class to the buttons that don't natively have icons and apply them through CSS.

**Additional context**
Pictures of CTA vs Button widget

![call-to-action-no-icon-option](https://user-images.githubusercontent.com/50156853/63134343-09fc3000-bf97-11e9-9474-841b58f50771.png)
![edit-button-has-icon-option](https://user-images.githubusercontent.com/50156853/63134344-09fc3000-bf97-11e9-8420-0cc50f951d12.png)

**Comment by vgstef** [comment]
Yeah, for buttons, we should at least be able to choose the size of the icon.

**Comment by bkjproductions** [comment]
+1
